src/components/Header/Header.tsx

```diff
@@ -0,0 +1,55 @@
+import { useContext } from 'react';
+import { TodosContext } from '../../contextes/TodosContext';
+
+const reset = () => ({
```

Comment:
why do you need a function for it?
Mb would it be better to make it like a variable?